### PR TITLE
Wdp200103 16

### PR DESCRIPTION
**Pull request by mariPab**
PROBLEM: Feature boxes were disordered at responsive view. 
SOLUTION: Bootstrap grid classes were added to display two boxes in one row on tablets and mobiles. On extra small devices boxes additionally receive fixed height and text inside them has smaller font size.
See two last commits (53174ce, e45d9a4).

### CODE REVIEW

File: src/components/common/ProductBox/ProductBoxContainer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import ProductBox from './ProductBox';
+import { getProductsToCompare, addToCompare } from '../../../redux/comparisonRedux';
+
+const mapStateToProps = state => ({
+  comparison: getProductsToCompare(state),
+});
+
+const mapDispatchToProps = (dispatch, props) => ({
+  addToCompare: () =>
```
Comment (konradkukier2) [inline src/components/common/ProductBox/ProductBoxContainer.js]: Tych zmian nie powinno być w tym PR

**Comment by konradkukier2** [comment]
Skoro w zakresie tego PR są tylko 2 ostatnie commity, to po co nam cała reszta? :P

**Comment by mariPab** [comment]
konflikty rozwiązane

**Comment by mariPab** [comment]
Utworzony nowy PR - Wdp200103 16 v2

**Pull request closed without merging**